src/server.h

```diff
@@ -2314,7 +2313,7 @@ typedef int redisGetKeysProc(struct redisCommand *cmd, robj **argv, int argc, ge
  *
  * The following additional flags are only used in order to put commands
  * in a specific ACL category. Commands can have multiple ACL categories.
- * See redis.conf for the exact meaning of each.
+ * See placeholderkv.conf for the exact meaning of each.
```

Comment:
Would be nice to refactor this information about categories out of the config file, which seems like an odd place, but probably not in scope here.